File: Python/1010.py
peca1 = input().split()
peca2 = input().split()

valor1 = float(peca1[-1])
valor2 = float(peca2[-1])

qnt1 = int(peca1[-2])
qnt2 = int(peca2[-2])

valor = (valor1*qnt1) + (valor2*qnt2)
print(f'VALOR A PAGAR: R$ {valor:.2f}')

File: Java/1010.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        List<Double> la = new ArrayList<>();
        List<Double> lb = new ArrayList<>();
        Scanner scanner = new Scanner(System.in);
        
        String a = scanner.nextLine();
        String[] na = a.split(" ");
        for (String numero : na) {
            double valor = Double.parseDouble(numero);
            la.add(valor);
        }

        String b = scanner.nextLine();
        String[] nb = b.split(" ");
        for (String numero : nb) {
            double valor = Double.parseDouble(numero);
            lb.add(valor);
        }

        double x = la.get(1) * la.get(2) + lb.get(1) * lb.get(2);
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", x);
    }
}